monkey/monkey_island/cc/services/reporting/report.py

```diff
@@ -381,37 +384,63 @@ def get_machine_aws_instance_id(hostname):
     def get_manual_monkey_hostnames():
         return [monkey["hostname"] for monkey in get_manual_monkeys()]
 
-    @staticmethod
-    def get_config_users():
-        return ConfigService.get_config_value(USER_LIST_PATH, True)
+    @classmethod
+    def get_config_users(cls):
+        usernames = []
+        configured_credentials = cls._credentials_repository.get_configured_credentials()
+        for credentials in configured_credentials:
+            usernames = chain(
+                usernames,
+                (
+                    identity
+                    for identity in credentials.identities
+                    if identity.credential_type == CredentialComponentType.USERNAME
+                ),
+            )
+        return [u.username for u in usernames]
 
-    @staticmethod
-    def get_config_passwords():
-        return ConfigService.get_config_value(PASSWORD_LIST_PATH, True)
+    @classmethod
+    def get_config_passwords(cls):
```

Comment:
Same thing, this is redundant to the credentials resource

Reply:
It's still needed because of `get_issue_set()` https://github.com/guardicore/monkey/blob/57b97d09f40671b8c455ab5d1d65d889548a481c/monkey/monkey_island/cc/services/reporting/report.py#L500, but we don't need to include the raw passwords in the response.